extensions/community_extensions/skadnetwork.md

```diff
@@ -82,30 +82,30 @@ If a DSP has at least one SKAdNetworkItem in the publisher app’s `Info.plist`
   <tbody>
     <tr>
       <td>
-        <code>version</code>
+        <code>versions</code>
       </td>
       <td>
-        Version of skadnetwork supported. Always "2.0" or higher. Dependent on both the OS version and the SDK version. </br></br><strong>Note</strong>: With the release of SKAdNetwork 2.1, this field is deprecated in favor of the <code>BidRequest.imp.ext.skadn.versions</code> to support an array of version numbers.
+        Array of strings containing the supported skadnetwork versions. Always "2.0" or higher. Dependent on both the OS version and the SDK version.
       </td>
       <td>
-        string
+        array of strings
       </td>
       <td>
-        "version": "2.0"
+        "versions": ["2.0", "2.1", "2.2", "3.0", "4.0"]
       </td>
     </tr>
     <tr>
       <td>
-        <code>versions</code>
+        <code>version</code>
       </td>
       <td>
-        Array of strings containing the supported skadnetwork versions. Always "2.0" or higher. Dependent on both the OS version and the SDK version.
+        Version of skadnetwork supported. Always "2.0" or higher. Dependent on both the OS version and the SDK version. </br></br><strong>Note</strong>: With the release of SKAdNetwork 2.1, this field is deprecated in favor of the <code>BidRequest.imp.ext.skadn.versions</code> to support an array of version numbers.
       </td>
       <td>
-        array of strings
+        string
       </td>
       <td>
-        "versions": ["2.0", "2.1", "2.2"]
+        "version": "2.0"
```

Comment:
not critical but i would say let's use "4.0" instead of "2.0" since this is the PR main goal

Reply:
Per the spec, `version` was deprecated once 2.1 was released and `versions` replaced it. I've not added 4.0 to this example as this field should not be used above 2.0.

Reply:
ok, got it

Reply:
There's a part of me that wonders if `version` can just be dropped from the extension to avoid confusion. Thoughts? Perhaps we just note it went away in a changelog.

Reply:
i think we should do it but it's not critical if we want to release it asap

Reply:
Perhaps we can keep but list with strikethroughs on all data in the row?